docs/source/index.rst

```diff
@@ -1,6 +1,13 @@
 Traits |version| Documentation
 ==============================
 
+Tutorial
+--------
+.. toctree::
+    :maxdepth: 3
+
+    traits_tutorial/index
+
```

Comment:
This being at the top makes it more likely the first thing a beginner would read. But I think the Introduction section is more suitable as the first thing to read because it provides a higher-level summary of the capabilities of Traits: https://docs.enthought.com/traits/traits_user_manual/intro.html#introduction

I'd think this tutorial needs to be read _after_ the introduction because it assumes one knows what Traits is for, is interested in it, and wanted to start learning to use it.

Once we establish a structure of our documentation to contain gentle guides/tutorials/API references/example snippets, it will likely get replicated to other ETS repository. So it might be worth thinking about this earlier. 

(I personally like sqlalchemy's documentation (e.g. [Version 1.3](https://docs.sqlalchemy.org/en/13/)). Its landing page guides you through an overview, installation, and then provide links to tutorial and API references.)